server/organization/tests.py

```diff
@@ -0,0 +1,60 @@
+import json
+from django.test import TestCase
+from django.test.client import Client as HttpClient
+from rest_framework import status
+from rest_framework.test import APITestCase
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APIClient
+from user_account.models import CustomUser
+
+
+class OpenRegistrationTestCase(APITestCase):
```

Comment:
This test case is dummy for an endpoint that will not be used in production. Maybe add these tests to the actual Registration endpoint test case class or create a new one just for organization ?

Reply:
it's for organization I just copied the structure but forgot to change the name of the class hahaha